contracts/uos.system/include/eosio.system/eosio.system.hpp

```diff
@@ -217,61 +297,186 @@ namespace eosiosystem {
       EOSLIB_SERIALIZE( voter_info, (owner)(proxy)(producers)(staked)(last_vote_weight)(proxied_vote_weight)(is_proxy)(flags1)(reserved2)(reserved3) )
    };
 
-    struct [[eosio::table, eosio::contract("uos.system")]] calc_voter_info {
-        name                owner;       /// the voter
-        std::vector<name>   calculators;     /// the calculators approved by this voter
+   /**
+    * Calculator Voter info.
+    *
+    * @details Stores information about the voter related to voting for Calculators:
+    * - `owner` the voter
+    * - `calculators` the calculators approved by this voter
+    */
+   struct calc_voter_info {
```

Comment:
changed from struct [[eosio::table, eosio::contract("uos.system")]] calc_voter_info to struct calc_voter_info {